labcodes/labstorybook
# New tokens

This PR aims to separate the `_tokens.scss` file into smaller scss files for colors, typography, spacing and animations (for now). We renamed the directory where they are from `tokens` to `variables` to match the recommended on the CSS [guidelines](https://paper.dropbox.com/doc/CSS-Guidelines-ZlPI0Lh6I38ZVFn8yY579)

We also created stories for animation and colors under menu item "atoms", and changed spacing and type system stories to be under the same "atoms" menu item.

**How to test:**
There isn't much to test. To see if all is working correctly, run the project and check if under "atoms" there are stories for animation, color, spacing and type system (animation stories is yet empty).

**Screenshots:**
![image](https://user-images.githubusercontent.com/5848493/83053217-cacf3500-a026-11ea-9e92-8d85cd9cb0ea.png)
